docs/index.rst

```diff
@@ -2,7 +2,7 @@
 Welcome to the sunraster Guide
 ******************************
 
-Welcome to the sunraster User Guide. sunraster is a SunPy-affiliated
+Welcome to the ``sunraster`` User Guide. ``Sunraster`` is a SunPy-affiliated
```

Comment:
```suggestion
Welcome to the ``sunraster`` User Guide. ``sunraster`` is a SunPy-affiliated
```

The official package name is lower case.  I know this looks silly at the start of a sentence though.